todo/tests/integration/test_tasks_delete.py

```
@@ -1,42 +1,43 @@
+        self.assertEqual(response.status_code, HTTPStatus.NO_CONTENT)
 
-    @patch("todo.repositories.task_repository.TaskRepository.delete_by_id")
-    def test_delete_task_not_found(self, mock_delete_by_id):
-        mock_delete_by_id.return_value = None
-        non_existent_id = str(ObjectId())
-        url = reverse("task_detail", args=[non_existent_id])
-        response = self.client.delete(url)
-        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
-        error_detail = response.data.get("errors", [{}])[0].get("detail")
-        self.assertEqual(error_detail, ApiErrors.TASK_NOT_FOUND.format(non_existent_id))
+    def test_delete_task_not_found(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)